### ISSUE TITLE
What can we improve from now on?

### ISSUE THREAD

**Issue by 6r1d** [rms-support-letter/rms-support-letter.github.io#6078]
Things are calming down, but we have two communities from now on. One denounces RMS, other doesn't.
There's still little communication and there's a bit of conflict when somebody doesn't want to listen to you.

The fact many don't know the situation can potentially hurt RMS at a later date.
FSF is almost broken, and it's probably broken because it relied on sponsorship money.
The fact our letter only had appeared only in some news is strange, as well.

What can we do from now on to mend and improve things? Should we act at all? I propose an open discussion on that matter.

**Comment by Qeenon** [comment]
I have suggestion: let close all our opensource projects (sorry in advance)

**Comment by 6r1d** [comment]
> (sorry in advance)

You don't have to be sorry for helping us think. Nobody should feel this way. There are too much reaction to the very discussion, so we get distracted and point fingers. And that hurts the community.

> I have suggestion: let close all our opensource projects

You have a point, actually. But that way helps the large companies, because it'll lead at worst to development of better disassembler and more closed source future. Or only one community that tends to ignore evidence.

**Comment by Qeenon** [comment]
I understand. That's my initiative and there is no alternatives for now. But that's something alive right now.

**Comment by 6r1d** [comment]
> I understand. That's my initiative

Yes, you decide.

> and there is no alternatives for now. But that's something alive right now.

Well, we have lots of good people with us. I think there are many things we can do.

**Comment by shenlebantongying** [comment]
The mission of this repo is completed. Bumping the number up is no longer impactful. The purpose is **not** to "fight over number". It's meaningless.

> There's still little communication

No, there are shit ton of communication already. Besides, the message already spread to the whole community.

Further things don't have to initiate from this repo. Just wait for the next move of FSF. 

Instead, we should stop accepting new signs as the open letter, so that we can do something else.

**Comment by 6r1d** [comment]
> The mission of this repo is completed. Bumping the number up is no longer impactful. The purpose is not to "fight over number" or "fighting between two parties". It's meaningless.

Huh, you have an interesting point. And you already told me something along the lines before.

> Further things don't have to initiate from this repo. Just wait for the next move of FSF.

Makes sense.

> Instead, we should stop accepting new signs as the open letter, so that we can do something else.

Honestly, it feels like a good idea to me, too.

**Comment by nukeop** [comment]
> Further things don't have to initiate from this repo. Just wait for the next move of FSF.
> 
> Instead, we should stop accepting new signs as the open letter, so that we can do something else.

Yes, I was thinking of closing the letter tomorrow, after merging the last batch of signatures from codeberg etc.
Mission accomplished.

**Comment by franpoli** [comment]
Unlike our opponents, as we close the letter, we should make a brief statement about our achievement. It would be good to end this action with an encouragement and unifying message.

**Comment by Tw1ddle** [comment]
Once signatures are no longer accepted, adding a closing statement above the letter that suggests other ways to provide support (e.g. making FSF donations or membership) could be constructive.

**Comment by shenlebantongying** [comment]
Maybe this thing on top of README?

![image](https://user-images.githubusercontent.com/20123683/113601252-153e1c80-960f-11eb-95df-8bb45d8ac1ff.png)

**Comment by BlueFlo0d** [comment]
> I have suggestion: let close all our opensource projects (sorry in advance)

I thought about another idea: relicense on our projects under a license that explicitly disallow or create trouble for MegaCorps. I think to do that, we just need such a clause: "You're free to use, reproduce or modify the software by any means, but any such usage is illegal" or so. It will be easy for anonymous hackers to develop and use such projects, but almost completely impossible for MegaCorps.

I don't know enough law to work out the full details. Any thoughts?

**Comment by 6r1d** [comment]
> I don't know enough law to work out the full details. Any thoughts?

While RMS was causing enough trouble for the corporations, I am not sure how much luck was involved.
Your approach will expand legal battles and we aren't an army of lawyers at the end of the day. I'm not sure if those are good or bad, though. If they'll allow to attract more attention to many real problems in software industry, so be it.

At this point, many of us had a wakeup call: seeing how many people or bots prevent important information from spreading was weird. Seeing how biased **platforms** are is weird, as well.

There are several routes we can take. One I would propose is to educate people. Not on RMS (who is very important), but more generally, how to start actually looking at evidence, how to treat evidence (critical thinking), gather enough of it, how to make theories from it (which is a tedious process people often replace with borrowed views called scientific method). If people knew even a part of it, none of what happened would happen. I hope.

And I hope we'll remain as a community to write code and react on weird stuff, too.

**Comment by shenlebantongying** [comment]
> One I would propose is to educate people.

Congrats, you just discovered how can we get world peace :)

**Comment by nukeop** [comment]
> I thought about another idea: relicense on our projects under a license that explicitly disallow or create trouble for MegaCorps.

It's called GPLv3 and AGPLv3.

**Comment by 6r1d** [comment]
> In theory, we can get world peace by doing that :)

I know it won't help immediately. But it'll help gradually. A tiny bit. Ish. We don't have a "fixus everythingus" spell, sadly.

**Comment by maxim-kr** [comment]
How could we make Organizations who signed letter to remove RMS. make responsible for their choice?  I want Organizations to feel that they are not as big and powerful to support everything "good" against everything "bad" without understanding the situation. I want them to think twice before doing that next time, because I don't want to live in the world where Apple or Microsoft tomorrow could publicly force my employer to fire me, because some hysterical person from their accounting department told them that I'm a bad guy.
If it would be some browser addon, that informs a user about their deсigion, when a user visits their website? But no one would install that addon. Those, who will install - they are already familiar with the situation. So how about a special list in already existing addons, that other people might already have, like UMatrix filter list? Maybe we need to write someone who already maintains sutable browser addon or filter list and who might want to support us?
Another option is to finf Mozilla and others on google maps, put one star, and leave the reply with a link to this letter and short explaining of situation. And don't forget about PlayMarket.
I bet, when Organization support would deal with 100-200 (maybe 1000) one-star replies with explaining the situation,. they would immidiately remove themselfs from that list and think twice next time. Should we try?

**Comment by maxim-kr** [comment]
My English is horrible, if someone could suggest small sharp clear message, clarifying the situation, I would leave it under every Organization software in PlayMrkt. So their customers could see it and become familiar with situation. 
What do you think on it?

**Comment by githubisnonfree** [comment]
> > Further things don't have to initiate from this repo. Just wait for the next move of FSF.
> > Instead, we should stop accepting new signs as the open letter, so that we can do something else.
> 
> Yes, I was thinking of closing the letter tomorrow, after merging the last batch of signatures from codeberg etc.
> Mission accomplished.

nukeop:

I think we should keep the petition going. Why settle for a lesser win?

Let's keep winning, forever. Keep the petition going!

**Comment by shenlebantongying** [comment]
It seems we can still get a few new signs this week.
![image](https://user-images.githubusercontent.com/20123683/113671527-77853480-9684-11eb-990a-b57194832d13.png)

**Comment by Qeenon** [comment]
I don't want to play this game if you think that "we win". It's not a war and we have lost already, and it's not a game.
FSF has lost support from many corporations and we are also "badly highlighted". (many of you don't care but I care, I personally has lost several friendly people already). You can also see that FSF has lost several important people who are not willing to work inside This anymore, you sure can still support FSF but people who was working many years and now reasonably leave will miss this support and there is no way back.

This letter was helping, hope giving, supporting and it's good thing. I agree with idea that it will be reasonable now to stop collecting signs/numbers because it was made visible already, corporations doesn't care (unless in bad way), community have seen that they are not alone and it will be smart to stop playing numbers.

Personally if there will be other movements against cancelling people for their views, gender, skin color (or whatever) - I'd be happy to join those (most likely) but it's not related to this repository. imho.

**Comment by shenlebantongying** [comment]
How about we keep it open until the Debian's vote? https://www.debian.org/vote/2021/vote_002

The proposal E is signing us and F is go against the open letter.

> Saturday 2021-04-17 23:59:59 UTC

**Comment by appetrosyan** [comment]
> (many of you don't care but I care, I personally has lost several friendly people already)

Wouldn't be too worried. They didn't care enough about you to give you a second chance and jumped to the conclusion that you're transphobic, and agreed to everything else said about RMS, without doing their own research. 

> I don't want to play this game if you think that "we win".

The split was already there before the open letter. People thought that way. People jumped to conclusions. 

You may have lost some friends in the process, but I'd argue you'd lose them over something else later on. Better now than later. The FSF isn't any worse than it was just before the open letter, it just revealed certain things.

**Comment by githubisnonfree** [comment]
nukeop, then how about leave it open until the end of the Debian vote?

Some other people have suggested that aswell. I think it's a reasonable end time. Ending the petition now would be too early, I think.

**Comment by skirpichev** [comment]
@nukeop, I urge you wait a little till the end of the Debian vote.  There is a little chance, that the project will sign our letter, yet it is.  Moreover, some DD's privately told me that they are ready to leave the project, like me, and join to your letter in case something like the "choice 1" will win.  Don't close the door!

**Comment by nukeop** [comment]
> nukeop, then how about leave it open until the end of the Debian vote?

We had a discussion with the team and we're ok with leaving it open until then. We're not expecting more than ~6-6.5k signatures in any event (which is already a huge success).

**Comment by skirpichev** [comment]
On Tue, Apr 06, 2021 at 04:04:50AM -0700, nukeop wrote:
>    We're not expecting more than ~6-6.5k signatures in any event (which
>    is already a huge success).

The point was: you didn't announce the deadline.  That's why debian
people include this letter in the vote, for example.

I think, given we know when voting period ends - now it's a good time
to provide some deadline.  Say "end of vote + 1 week".

**Comment by githubisnonfree** [comment]
nukeop, so I suggest leaving the letter open for 5 additional days after the debian vote is announced.

I have a feeling that they might be supportive of us. It's worth waiting.

**Comment by basilean** [comment]
Hello community,
 Please keep this letter open, as me, many people will get news on late and will find this place comfortable sharing the feeling.

 Personally, I think to exclude Stallman from FSF is like to exclude Trotski from Communist party and we all know how the Soviet Union ended.

 Its pretty clear that corporation lackeys are producing this fracture, its not just about software, its far deeper.

Thank you very much for this place, make you feel that community is alive and the sense of solidarity still burns.

**Comment by cmpunches** [comment]
> The mission of this repo is completed. Bumping the number up is no longer impactful. The purpose is **not** to "fight over number" or "fighting between two parties". It's meaningless.
> 
> >

No it is absolutely not completed.

You need at least 10k signatures to beat this forever.

If you close the repo before reaching 10k signatures your adversaries will suppress and mitigate the existence of this and effectively waste all of your effort.  They are trained PR professionals.

**Comment by majestrate** [comment]
I think that the profound success of the support letter being able to scale submissions compared to the open letter who have collapsed from an impractical project management structure says everything that needs to be said. This project here is a well oiled machine and the open letter seems to have ceased operation for the most part. 

It's also quite regrettable that so many prominent project leads opposed the community will at large, it really shows how either blatantly out of touch they are with their demographics or how pervasive the corporate influence is.

**Comment by 6r1d** [comment]
> Please keep this letter open, as me, many people will get news on late and will find this place comfortable sharing the feeling.
> No it is absolutely not completed.

Yes, there's a very high possibility the letter will stay open. :-)

> This project here is a well oiled machine and the open letter seems to have ceased operation for the most part.

[Bryan Lunduke](https://odysee.com/@Lunduke:e) **[intends to communicate with open letter people](https://odysee.com/@Lunduke:e/TakingAStand:b)**. Not as a signer, but on his own behalf. Maybe future is brighter: he's well-known.

Let's watch how it happens and support him in comments while he's telling about the progress.

**Comment by nukeop** [comment]
We will keep the letter open indefinitely because of Richard Stallman's request I received by email:

```
Please don't close it to signatures.

If you assume that "the whole community has seen it," that conclusion
would follow.  But there is no "whole community" and news does not
spread that fast.

New people will continue to hear about it and some will sign.  If 100
people per month sign, it will be good to have those signatures.
```

**Comment by Shamar** [comment]
Hi guys, sorry for joining this discussion a bit late, but I was occupied on [other fronts](https://gcc.gnu.org/pipermail/gcc/2021-April/235285.html).

I agree that fighting over numbers is pointless and that closing the letter after Debian's is a good idea even just because otherwise the attackers might argue that the comparison with their failure is unfair.

Also note that they closed the harrassing letter after several people removed their own signatures and after having to retract [several lies in their appendix](https://rms-open-letter.github.io/appendix) to reduce legal risk for the signers (defamation is a crime in many countries).

IMHO, if there something I feel we should celebrate as a win, it's not numbers but these two facts.

Sure, all of this uncovered a huge issue in Free Software: the embrace of BigTech is a deadly one.
It's marginalizing Free Software hackers and corrupting and colonizing (even remotely) successful projects [with the Open Source rhetorics](https://medium.com/@giacomo_59737/what-i-wish-i-knew-before-contributing-to-open-source-dd63acd20696).

I think we should really do something about that.

> relicense on our projects under a license that explicitly disallow or create trouble for MegaCorps.

> Your approach will expand legal battles and we aren't an army of lawyers at the end of the day. I'm not sure if those are good or bad, though. If they'll allow to attract more attention to many real problems in software industry, so be it.

I'm personally working on this since years with the [Hacking License](http://www.tesio.it/documents/HACK.txt) (still work in progress) whose goal is to become the strongest possible copyleft with the widest reach and strongest reciprocity, so that nobody could adopt embrace, extend and extinguish tactics or prevent self-hosting.

Another interesting approach is the [Cryptographic Autonomy License](https://opensource.org/licenses/CAL-1.0), but since it doesn't share copyright with the users that respect the license, I still think we can do better.

The goal is not to prevent any use, but to prevent all abuses that corporations do to Free Software.

I think this is a debate we should have.
(arguably not here on GitHub)

If you are interested, feel free to contact me.

**Comment by skirpichev** [comment]
> The goal is not to prevent any use, but to prevent all abuses that corporations do to Free Software.

Sorry, you can't.  It's capitalism.

**Comment by Shamar** [comment]
You wrote:

> Sorry, you can't. It's capitalism.

But I read:

> Sorry, you can't. He's the **Pharaoh**.

There is no greater victory for an oppressor, than the oppressed believing they cannot beat the oppressing system.

**Comment by Shamar** [comment]
> We will keep the letter open indefinitely because of Richard Stallman's request

Sorry I had missed that post before. Fine, I trust RMS's perspective on this.

**Comment by skirpichev** [comment]
> But I read

@Shamar, you should read literally.

> There is no greater victory for an oppressor, than the oppressed believing they cannot beat the oppressing system.

Software market is a very tiny part of that system.  And not a critical one, unless you can eat bytes.

RMS started the FOSS movement as a something, that fits the currents US eco

**Comment by githubisnonfree** [comment]
btw i'm sure you know, stallman.org now links to stallmansupport.org which links to the petition

i suggest keeping the petition going for at least a few more weeks, if not longer

this battle isn't over. i think the anti-rms crowd is planning something. i can't prove that, but it's a gut feeling.

**Comment by 6r1d** [comment]
@Shamar, thank you for a big work related to GCC steering commitee discussion. It matters quite a bit and I agree with your points.

> If you are interested, feel free to contact me.

I am certainly interested. Don't be surprised by an email. :-)

**Comment by CarlosEkisde** [comment]
Now it's time to start donating. To win this, or to survive. We need to put all the effort we can. Big Tech is perverting free software projects to putting them at the service of their evil purposes. They fear RMS and FSF because they remind us that beyond the scope of creating software with the maximum technical capabilities, exists the right of the user to be free.  And this definitely clashes with what they want in the future for us.

**Comment by emestee** [comment]
> I agree that fighting over numbers is pointless and that closing the letter after Debian's is a good idea even just because otherwise the attackers might argue that the comparison with their failure is unfair.

One could argue that the opposing letter closed for signatures after realizing that they're looking at a defeat. 

> I think we should really do something about that.

There are several principles that I can think of that can be contemplated, but would certainly rile people up.

1. Any individual in a voting or directorship role in any public organization such as the FSF or the GCC SC who is also employed by a large corporation is an automatic conflict of interest. There are no ifs and buts. At the very least such individuals should affirm in public that their views and opinions are their own, and that they understand it would be a breach of trust for them to use their position in any manner to advocate the political, commercial and technological interests of their employer (_cough_ systemd _cough_)
2. Outside of the narrow scope of interests of digital liberties, copyright and online censorship, public projects should not take stances on political issues, because doing so divides the contributors and attracts angry mobs. It follows that public projects can not spare their resources to execute political advocacy on behalf of others, e.g. it is not appropriate for the GNU Project to advocate for or against guns, abortions, drug legalization, illegal immigration, taxation and so on, as none of those issues pertain to free software. Members should be free to do so on their own time, without fear of retribution, as is their right, which means the orwellian codes of conduct must go.
3. Similarly, it is not appropriate for public projects to devote their resources to activism that is unrelated to the narrow scope of interests, which means public projects should not engage in any sort of diversity initiatives, sponsor events that have political colors (as FSF did with SeaGL 2020, which hosted der.hans, Hashman, Nicholson, de Blanc, McGovern and Brasseur, all of whom are now mounting an assault on FSF - yes, these people are all in the same conference boat), or signal any support for any unrelated political movements, regardless of these movements' virtues or goals.
4. The value of meritocracy, or more specifically technocracy, must be reemphasized, in that contributions are judged solely on their merit, and not on any individual attributes of their authors, ranging from color of skin to political views and twitter feed contents.
  Organizations such as FSF need to develop an immunity from outside influences, a

**Comment by 6r1d** [comment]
> One could argue that the opposing letter closed for signatures after realizing that they're looking at a defeat.

Well, let them enjoy counting the big names (who are probably misled quite often)

> cough systemd cough

Cough indeed!

> Any individual in a voting or directorship role in any public organization such as the FSF or the GCC SC who is also employed by a large corporation is an automatic conflict of interest

Current GCC steering commitee has [quite a few of non-corporate names](https://gcc.gnu.org/steering.html). It seems like a perfect group to decide a fate of the most used (and useful) compiler! ;-)

> which means the orwellian codes of conduct must go

All codes of conduct should. I already saw some discussion where someone told those are the rules to ban people for. Guess they enjoy too many people who are eager to contribute. Does GitHub require code of conduct for donations? If such hooks are everywhere, they have to go. Or better alternative platforms will appear.

> Similarly, it is not appropriate for public projects to devote their resources to activism that is unrelated to the narrow scope of interests

...Or the software will stagnate.

> The value of meritocracy, or more specifically technocracy, must be reemphasized

Good idea

> immunity from outside influences, a

That was rather abrupt. Is everything good?

**Comment by emestee** [comment]
I accidentally pressed S-Enter, I edited the last sentence after that.

**Comment by Aspie96** [comment]
I am of the strong opinion that the initiative should be closed.
This is for a few reasons:

1) We are not getting that many signatures.
2) By closing the letter we can announce a defenitive result with clear numbers
3) In the long run the two letters will no longer be comparable. If the other letter is closed and this one isn't, new votes will mean nothing. Of course we only will be getting votes, but it will no longer be a response to the other letter.
4) There could be future controversies about Stallman, or anyone else. I think this letter should only be referring to the current situation as a response to the open letter. This is better rapresented if the letter is closed soon.

Note that:
1) As of now, the two letters are comparable. It's true, we kept it open longer, but it's ok. They closed it because they were getting fewer signatures. Today, even including the signatures they added after closing the letter, they have fewer signatures than we had the day they closed the letter. This is significant. Also, if you look at the GitHub stars (which are NOT a formal way of voting and should not be. We should not call for stars in order to keep this symmetrical) it's clear which letter is preferred by the community. But we should not keep it open too long or else we will not be able to announce a comparable result.
2) Of course the letter must remain open forever for those wishing to remove signatures.

Note: I am aware that Stallman prefers keeping the letter open and I am openly disagreing with him

**Comment by Aspie96** [comment]
What we want to do in the future is to support the FSF by donating and asking others to donate and to support Free Software.

But the battle between this two letters should not continue. We won: it's now a useless race against a dead body.

**Comment by cmpunches** [comment]
> 3\. In the long run the two letters will no longer be comparable. If the other letter is closed and this one isn't, new votes will mean nothing. Of course we only will be getting votes, but it will no longer be a response to the other letter.

@Aspie96 Strategically this doesn't make as much sense as it may seem to at first glance.  There is some wisdom in avoiding a reactionary model in an adversarial situation.  By tying the actions for our initiatives so tightly to the actions of the exclusionist camp, we inadvertantly put ourselves in the position of being controlled by an external actor.

> * it's clear which letter is preferred by the community. But we should not keep it open too long or else we will not be able to announce a comparable result.

We should treat the existence of the other letter as a separate issue entirely.

> 3\. If they open the letter ag[a]in we can too.

Sure, but, bear in mind the exclusionist camp has paid PR teams and is a learning, adaptive organization and a high level of (misguided) motivation.  Even while we've been focused on this they have been trying to subvert its existence in the press so that they can recover enough from the blowback of a failed campaign to go for round 2.  This was the easiest of all of their efforts because it was the first one.  The only solution is a comprehensive strategy of which this is one part, so, we need this to stay alive in perpetuity.

**Comment by Aspie96** [comment]
> @Aspie96 Strategically this doesn't make as much sense as it may seem to at first glance. There is some wisdom in avoiding a reactionary model in an adversarial situation. By tying the actions for our initiatives so tightly to the actions of the exclusionist camp, we inadvertantly put ourselves in the position of being controlled by an external actor.

This is a good point.

However, you have to consider this letter is simply a response to theirs. So our actions will be tied to theirs. And, more importantly, they are tied to this controversy, not any future one

It's true we have to think about our decisions, but I think after considering all factors we should close this letter soon. It would not have been wise to close it when they closed theirs beause they were already not receiving signaturse whyile we were still receiving many.

> We should treat the existence of the other letter as a separate issue entirely.

I disagree.

I think this letter is clearly a response to the other letter, or at least by the movement which is represnted by it

> The only solution is a comprehensive strategy of which this is one part, so, we need this to stay alive in perpetuity.

I do not think at all keeping it open is the best strategy.
We can reopen it as soon as they do and I don't respect a different result if we act similarly.

Indeed, I think this is precisely a good reason to close the letter for two reasons:
1) I think opening it again (when needed, which is to say after they open it) and starting a new separate call for signatures actually could be more effective than keeping it open since it will be more noticible
2) If we keep the letter open those who haven't signed it yet and want to will. Imagine that then they open the other letter with a new successful call for signatures. Those who have signed our letter in the meanwhile will obviously not sign it again, but they will be people who would have been convinced later too, therefore if we keep it open it will actually be harder to have a strong and visible wave of new signatures. Will we have many more signatures in total? Yes. But they will claim those signatures mean nothing and do not represent what happened after their new call for signatures because indeed they will have gotten more signatures after the new call and be getting more signatures faster

**Comment by 6r1d** [comment]
Reminder: this exists. Voting stops in less than an hour. There's a high chance Debian will stay neutral and some devs will join this letter.

* https://www.debian.org/vote/2021/vote_002
* https://vote.debian.org/~secretary/gr_rms/vote_detailed.html

-----

Generally I don't have a strong personal opinion about opening or closing. I wanted to close it some time ago and I was wrong then: some interesting people joined only recently.

**Comment by Aspie96** [comment]
I am not suggesting closing it before Debian votes.

Indeed, I think closing ought to be announced (and at minimum it should have been after Debian's vote even if we could announce it to the past).

I am suggesting it should be closed in few days, defenitely not right now and surely not prior to Debian's vote.

Indeed closing it can actually get us a wave of signature making sure lazy people sign too

**Comment by cmpunches** [comment]
> Indeed closing it can actually get us a wave of signature making sure lazy people sign too

So on that I agree.  Even as far out as 90 degrees, the announcement of the signature window closing would be great marketing pressure to bring in another spike of voters.  I was hoping to see this reach 10k signatures before closing.

**Comment by cmpunches** [comment]
Look, regarding Debian, there are two entities at play here:  

1) There's Debian the organization, which has clearly been hijacked by corporate interests using identity politics as a weapon to attack areas that those interests have declared as an existential enemy way before they became foss sponsors.

2) There's also the Debian community, which is going to have believers of the multiple "realities" at play there.  Some of them are allies.  Some of them are not.

That spike can only be expected from (2).

**Comment by Aspie96** [comment]
I think considering my other points 90 days is probably too long.

I don't think we should focus an arbitrary number based on our arbitrary base 10 system but, rather, a more high level result.

Our closing time should balance a few factors which should include not waiting too long after they closed their letter and, also, the rate at which we are currently getting signatures.

We should not be trying to maximize in number of signatures but, rather, in the strenght of the message sent.
I think we have mostly sent our message and indeed won. Closing signatures makes such win clearer IMO and makes it actually easier, not harder, to get new signatures faster than them if they open their letter a second time. It also avoids confusing situations in case a new controversy arises.

Obviously PRs themselves should remain open (or at least there should be a clear separate way for removing signatures)

**Comment by 6r1d** [comment]
> That spike can only be expected from (2)

Nah, I care about more widespread reaction when Debian announces something. At all. Can we just watch what will happen for now?

**Comment by Aspie96** [comment]
I think that by closing signatures the message will be much clearer expecially in a distant future: "there were 2 initiatives, one got A signatures, the other got B signatures, so B won", rather then "there were two initiatives, one got A signatures, the other is still collecting signatures".

What is important now is to support the FSF and to create awareness but without focusing on this letter, which at this point is a distraction
And we must fight against any such cancel mob we see, not just this one. But I think we safely won this one specific battle.

**Comment by cmpunches** [comment]
> I think that by closing signatures the message will be much clearer expecially in a distant future: "there were 2 initiatives, one got A signatures, the other got B signatures, so B won", rather then "there were two initiatives, one got A signatures, the other is still collecting signatures".
> 
> What is important now is to support the FSF and to create awareness but without focusing on this letter, which at this point is a distraction
> And we must fight against any such cancel mob we see, not just this one. But I think we safely won this one specific battle.

I yield my dissent to this idea.

**Comment by cmpunches** [comment]
It looks like their vote has concluded.  They've unfortunately convoluted their election process so much that the results are unreadable without further research.

**Comment by Aspie96** [comment]
Their decision is to be neutral

**Comment by cmpunches** [comment]
> Their decision is to be neutral

Glad to hear that's what Debian the organization has voted on, but it doesn't detract from the fact that this was Neil McGovern's campaign.

It really should be addressed.

**Comment by franpoli** [comment]
Indeed, Debian's vote is difficult to interpret. [According the pool result](https://vote.debian.org/~secretary/gr_rms/results.txt), they choosed the option 7 which states that "Debian will not issue a public statement on this issue". However [the full statement of proposal 7](https://www.debian.org/vote/2021/vote_002#statistics) was: 

```
Choice 7: Debian will not issue a public statement on this issue
The Debian Project will not issue a public statement on whether Richard Stallman should be removed from leadership positions or not.

Any individual (including Debian members) wishing to (co-)sign any of the open letters on this subject is invited to do this in a personal capacity.
```

This last sentence does not honor the Debian signatories of the petition who led the offensive against the FSF before their internal consultation.

**Comment by Aspie96** [comment]
> This last sentence does not honor the Debian signatories of the petition who led the offensive against the FSF before their internal consultation.

I don't understand your point actually.

**Comment by cmpunches** [comment]
> I don't understand your point actually.

@Aspie96 The smear campaign that this all revolves around was led by a guy named Neil McGovern, who works at Debian and the Gnome Foundation, and Elana Hashman who works at Debian and the Kubernetes project, and Molly de Blanc at Debian and Gnome foundation was in all likelihood set up as a patsy to point the public's anger at to allow Neil and Elana to continue doing things like this.

The proposal that was on debian's docket here for joining in on attacking the FSF was also brought forth by Neil.  He was also involved in the media arm of the effort where he went onto podcasts to try to suppress the support letter's results.

He is like the Final Boss of identity politics trolls.

**Comment by franpoli** [comment]
1. Debian did not explicitly followed in the offensive against the FSF
2. Debian "authorize" its members to sign any open letters on their own capacity (self-deecision).

These are two forms of disavowal against Molly de Blanc, Elana Hashman, Neil McGovern and Stefano Zacchiroli who have been proud to affiliate themself to Debian in the smear letter.

**Comment by Aspie96** [comment]
> These are two forms of disavowal against Molly de Blanc, Elana Hashman, Neil McGovern and Stefano Zacchiroli who have been proud to affiliate themselves to Debian in the smear letter.

They are only signing in their individual capacity

**Comment by franpoli** [comment]
@Aspie96 They marked their affiliation to Debian, some use a Debian email address, some were part of the Debian vote. This tells about the respect these person have for the Debian community.

**Comment by cmpunches** [comment]
> > These are two forms of disavowal against Molly de Blanc, Elana Hashman, Neil McGovern and Stefano Zacchiroli who have been proud to affiliate themselves to Debian in the smear letter.
> 
> They are only signing in their individual capacity

@Aspie96 no they didn't just sign, they literally were the coordinators of this attack.

**Comment by Aspie96** [comment]
Yes, but, again, in their individual capacity. Not representing Debian.
The official stance of Debian is neutral and there is no contraddiction there.

**Comment by 6r1d** [comment]
If it's about Debian now, maybe it's time to use [GitHub discussions](https://github.com/rms-support-letter/rms-support-letter.github.io/discussions)? :-)

I made this issue for *improvement proposals*. So some *possible improvements* are proposed *here*.

**Comment by franpoli** [comment]
> Yes, but, again, in their individual capacity. Not representing Debian.
> The official stance of Debian is neutral and there is no contraddiction there.

@Aspie96 I'd like to make it clear. I am not passing judgement on Debian. I am pointing out the methods of manipulation used by jerks who seek to defame us.

**Comment by cmpunches** [comment]
> > Yes, but, again, in their individual capacity. Not representing Debian.
> > The official stance of Debian is neutral and there is no contraddiction there.
> 
> @Aspie96 I'd like to make it clear. I am not passing judgement on Debian. I am pointing out the methods of manipulation used by jerks who seek to defame us.

I reiterate what @6r1d said about this meriting a discussions thread, but I think this comment should set it off:  All these people work for Debian and hover around the OSI.  I do not believe it would be reasonable to conclude that this is a coincidence, but, regardless, the behavior of these 3 people should be addressed in a community sense.  Literally anyone could be their next victim -- including people in their specific communities.

**Comment by basilean** [comment]
Hello community,
 I will share my thoughts and auto critic.
 I have been using GNU/Linux Debian for the last 20 years, I remember to read Ian's and Richard's manifestos like a call to arms to defend my freedom. It has been so sad these past weeks, what went wrong that community flagship is at risk to be taken by  [haters mob | company lackeys]? even worse, one of them was running for leader (check proposal 2) with a platform that looks like a copied and pasted from a big company newsletter. Not talking about disappointment to see proposals supporting FSF with minimal votes.
 From my side, I thought it was enough to contribute in the community not being part of the project but nowadays I'm feeling the need to join and try to build a new generation that remember the roots handling these challenges.
 I hope this experience will wake up many of us to defend the ship.

**Comment by 6r1d** [comment]
> I remember to read Ian's and Richard's manifestos like a call to arms to defend my freedom

Indeed, the attack vector was one of the best surprises to me as well. But @basilean, I would say right now: it is not the last **mob** attack. More will happen. More leaders will be attacked and removed. Maybe just programmers, too.

We all should stand against the mob tactic, against the idea someone can lose their life's work under false accusations and a lack of arguments.

**Comment by Aspie96** [comment]
> We all should stand against the mob tactic, against the idea someone can lose their life's work under false accusations and a lack of arguments.

I fully agree.
